packages/frontend/src/components/campaigns/index.tsx

```diff
@@ -208,11 +212,45 @@ export function Campaigns() {
     function handlePage(page: number) {
         setPageNumber(page);
     }
-
     const handleCreateCampaign = useCallback(() => {
         localizedRouter.push("/campaigns/create");
     }, [localizedRouter]);
 
+    const statusSelectRenderOption = (option: {
+        label: string;
+        color?: string;
+    }) => {
+        return (
+            <div className={styles.customOptionContainer}>
+                {option.label.toLowerCase() !== "all" && (
```

Comment:
We can use the `value` here for the condition, no?
```tsx
option.value === FilterableStatus.All
```

Reply:
I tried that but I don't remember I faced some issues with that. I'll try again and let you know